### PR TITLE
Addressing Delivery feedback from the field - ALZ

### PR DESCRIPTION
**Pull request by tsc-buddy**
Things addressed in this PR are as follows:

- Put the design areas in the correct order, both within the JSON and via new ID Column
- Address a number of Typos and wording issues
- Sorted the management sub category IDs out
- Renamed AAD to Microsoft Entra ID

### CODE REVIEW

File: checklists/alz_checklist.en.json [hunk 1]
```diff
@@ -480,359 +357,230 @@
...
-      "category": "Management",
-      "subcategory": "Monitoring",
-      "text": "Is the landing zone documented?",
-      "waf": "Operations",
-      "guid": "e179b599-de0d-4597-9cd4-cd21b088137f",
-      "id": "D03.02",
-      "severity": "Medium"
+      "category": "Resource Organization",
+      "subcategory": "Subscriptions",
+      "text": "Enforce a dedicated connectivity subscription in the Platform management group to host an Azure Virtual WAN hub, private Domain Name System (DNS), ExpressRoute circuit, and other networking resources.",
```
[the remark this answers is not in the document]
Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:446]: Updated in latest commit.

File: checklists/alz_checklist.en.json [hunk 2]
```diff
@@ -1096,7 +844,7 @@
       "text": "Ensure that shared networking services, including ExpressRoute gateways, VPN gateways, and Azure Firewall or partner NVAs in the central-hub virtual network. If necessary, also deploy Active Directory domain controllers and DNS servers.",
```
[the remark this answers is not in the document]
Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:844]: Updated in latest commit.

File: checklists/alz_checklist.en.json [hunk 3]
```diff
@@ -1987,333 +1735,461 @@
...
-      "text": "Aim to define functions for Azure Landing Zone Platform team.",
-      "waf": "Operations",
-      "guid": "634146bf-7085-4419-a7b5-f96d2726f6da",
-      "id": "H01.02",
+      "category": "Governance",
+      "subcategory": "Governance",
+      "text": "Identify required Azure tags and use the 'append' policy mode to enforce usage.",
+      "waf": "Security",
+      "guid": "e979377b-cdb3-4751-ab2a-b13ada6e55d7",
+      "id": "E01.02",
```
[the remark this answers is not in the document]
Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:1759]: Updated in latest commit.

File: checklists/alz_checklist.en.json [hunk 4]
```diff
@@ -1987,333 +1735,461 @@
...
     {
-      "category": "Resource Organization",
-      "subcategory": "Subscriptions",
-      "text": "Ensure tags are used for billing and cost management",
-      "waf": "Security",
-      "guid": "5de32c19-9248-4160-9d5d-1e4e614658d3",
-      "id": "I02.16",
+      "category": "Management ",
+      "subcategory": "Scalability",
+      "text": "Leverage Azure Virtual Machine Scale sets to scale up and down based on the load.",
```
[the remark this answers is not in the document]
Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:2179]: Updated in latest commit.

File: checklists/alz_checklist.en.json [hunk 5]
```diff
@@ -480,359 +357,230 @@
...
     {
-      "category": "Management ",
-      "subcategory": "Fault Tolerance",
-      "text": "Azure Load Balancer and Application Gateway distribute incoming network traffic across multiple resources.",
-      "waf": "Reliability",
-      "guid": "84101f59-1941-4195-a270-e28034290e3a",
-      "id": "E01.03",
+      "category": "Resource Organization",
+      "subcategory": "Subscriptions",
+      "text": "If AD on Windows Server, establish a dedicated identity subscription in the Platform management group to host Windows Server Active Directory domain controllers",
```
[the remark this answers is not in the document]
Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:556]: Updated in latest commit.

File: checklists/alz_checklist.en.json [hunk 6]
```diff
@@ -1987,333 +1735,461 @@
...
+      "waf": "Security",
+      "guid": "d8a2adb1-17d6-4326-af62-5ca44e5695f2",
+      "id": "E01.03",
+      "severity": "Medium",
+      "link": "https://learn.microsoft.com/azure/governance/policy/overview"
+    },
+    {
+      "category": "Governance",
+      "subcategory": "Governance",
+      "text": "Establish Azure Policy definitions at the top-level root management group so that they can be assigned at inherited scopes",
```
[the remark this answers is not in the document]
Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:1776]: Updated in latest commit.

File: checklists/alz_checklist.en.json [hunk 7]
```diff
@@ -406,25 +406,13 @@
       "training": "https://learn.microsoft.com/learn/modules/azure-architecture-fundamentals/",
       "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/resource-org-management-groups"
     },
-    {
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:530]: @tsc-buddy I had a question on this - to confirm, we are wanting to remove the recommendation to do tagging through policy?

Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:530]: Hey @brsteph - great question, the feedback we received was that it was a double to the policy with the following GUID: e979377b-cdb3-4751-ab2a-b13ada6e55d7

 "text": "Enforce or appended resource tags through Azure Policy",
&
"text": "Identify required Azure tags and use the 'append' policy mode to enforce usage."

Comment (tsc-buddy) [reply checklists/alz_checklist.en.json:530]: Have tweaked the wording to reference azure policy on the remaining item. Let us know if you are happy with that?

Comment (brsteph) [reply checklists/alz_checklist.en.json:530]: Looks good, thanks for the clarification!

**Review by igorjnzl — APPROVED** [review]
all feedback resolved

**Pull request merged**